scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SchedulerPage.java

```diff
@@ -90,6 +82,19 @@
  */
 public class SchedulerPage implements SchedulerStatusListener, LogListener, ExecutionDisplayModeListener {
 
+    /** Actions on the scheduler */
+    private final static String START = "Start";
+
+    private final static String STOP = "Stop";
+
+    private final static String FREEZE = "Freeze";
+
+    private final static String RESUME = "Resume";
+
+    private final static String PAUSE = "Pause";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reorder the modifiers to comply with the Java Language Specification. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AModifiersOrderCheck)